oauthproxy.go

```diff
@@ -989,6 +1028,8 @@ func (p *OAuthProxy) addHeadersForProxying(rw http.ResponseWriter, req *http.Req
 // CheckBasicAuth checks the requests Authorization header for basic auth
 // credentials and authenticates these against the proxies HtpasswdFile
 func (p *OAuthProxy) CheckBasicAuth(req *http.Request) (*sessionsapi.SessionState, error) {
+	client := p.GetRemoteAddr(req)
```

Comment:
I don't love the inconsistency of `client` and `remoteAddr` here.  I kept it this like to stick with the naming in the `logger` package, but maybe a rename is in order.